### ISSUE TITLE
Problematic implementation of Scene control

### ISSUE THREAD

**Issue by meganvilla** [OpenSpace/OpenSpace#3114]
The "Scene" control triggers the display of a left-side panel with datasets and other  elements. It is currently inaccessible to many groups of users: 
- Keyboard-only users can reach the control due to the tabindex attribute, but they  cannot activate it using Enter or Space. 
- It has a partial tab implementation, just as the "Settings" custom control which has a similar functionality. However, this means that screen reader users will hear incorrect details about the component. 

  - Although it has role="tab", it is not part of a tablist element, meaning that  users are not informed of the number of items in the set. 
  - The selected control is not communicated as such, although visually this is indicated using a blue bottom border and slightly lighter background color.  
  - The controlled panel is not marked up as a tab panel. 
  - The tabs keyboard navigation pattern is not implemented, and arrow keys  cannot be used to navigate between different tabs. 

- Note that although "Scene" and "Settings" are styled to look as tabs, they do not  function as a Tabs widget since: 
  - By default no tab is selected. 
  - When a tab is selected, the panel is displayed, and the cursor can be noticed in the "Search" field to indicate that the field is active. 
  - Users cannot activate a tab to close the corresponding panel. 

**Suggested Solution** 
Implement "Scene" using a native button element and remove role="tab" and  tabindex="0". Use aria-pressed to communicate the state of the "Scenes"/"Settings"  buttons, with a value of "false" when the corresponding menu pane is hidden and "true"  when displayed.

Note that it has been previously suggested to group the buttons at the bottom of the  screen in one or two toolbar components, depending on how they relate to each other.  See: 
- [Missing Keyboard Navigation Mechanism ](https://github.com/OpenSpace/OpenSpace/issues/3104)
- [Problematic Implementation of Landmark Region ](https://github.com/OpenSpace/OpenSpace/issues/3106)

Grouping only "Scene" and "Settings" may potentially be confusing since one is  intended to make changes to the application in general while the second has an effect  on the object in focus in the space viewer area.

Severity: 1

**Comment by WeirdRubberDuck** [comment]
Closed in #3608 

The Scene menu is now a window just like any other
